### ISSUE TITLE
When the curse from your weapon is lifted

### ISSUE THREAD

**Issue by eyestalker** [jeremygurr/dcssca#200]
due to the curse-expiring mechanic, the interface isn't updated properly and still shows it as cursed.

**Comment by jeremygurr** [comment]
Yeah I've been working on this, but haven't figured it out yet. It's some kind of webtiles caching thing I think.
